interface/forms/newpatient/common.php

```diff
@@ -223,6 +223,16 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
                   continue;
               }
 
+              // Fetch acl for category of given encounter. Only if has write auth for a category, then can create an encounter of that category.
+              $postCalendarCategoryACO = fetchPostCalendarCategoryACO($catId);
+              $postCalendarCategoryACO = explode('|',$postCalendarCategoryACO);
```

Comment:
Not a big deal but I think you can simplify a bit with:
$postCalendarCategoryACO = explode('|',fetchPostCalendarCategoryACO($catId));